plugins/org.jkiss.dbeaver.ext.sqlite/src/org/jkiss/dbeaver/ext/sqlite/model/SQLiteMetaModel.java

```diff
@@ -217,6 +217,8 @@ public boolean handleSequenceCacheReadingError(Exception error) {
 
     @Override
     public GenericTableColumn createTableColumnImpl(@NotNull DBRProgressMonitor monitor, JDBCResultSet dbResult, @NotNull GenericTableBase table, String columnName, String typeName, int valueType, int sourceType, int ordinalPos, long columnSize, long charLength, Integer scale, Integer precision, int radix, boolean notNull, String remarks, String defaultValue, boolean autoIncrement, boolean autoGenerated) {
+        typeName = (typeName == "N/A") ? "ANY" : typeName; //$NON-NLS-1$
```

Comment:
Did you test if this even works? In Java, you can't compare strings using `==`. You must use `.equals`.

Reply:
By the way, where does that `N/A` come from? I'd fix it there, not here.

Reply:
yes, it works... I have seen the codacy issues and correct them this evening.

The "N/A" come from the GenericMetaModel class :
https://github.com/dbeaver/dbeaver/blob/363c07ab6b023e73bbb53fd91039c44526e89161/plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/model/meta/GenericMetaModel.java#L773C1-L785C10

You're right, I can create a getDefaultTypeName() method in the GenericMetaModel, overrided in the SQLiteMetaModel class...

Reply:
I made these changes